.github/workflows/twig-cs-fixer.yml

```
@@ -0,0 +1,50 @@
+name: Twig CS Fixer
+
+on:
+  push:
+    paths:
+      - '**.twig'
+  pull_request:
+    paths:
+      - '**.twig'
+
+concurrency:
+    group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+    cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  twig-cs-fixer:
+    name: Twig CS Fixer
+    runs-on: ubuntu-24.04
+
+    env:
+      php-version: '8.4'
+    steps:
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ env.php-version }}
+          ini-values: "memory_limit=-1"
+          coverage: none
+
+      - name: Checkout target branch
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.base_ref }}
+
+      - name: Checkout PR
```

```suggestion
      - name: Checkout
```
This action seems redundant, since the target branch is not used in this case.

---

indeed, left over of copy/paste